### PR TITLE
Baijun 33 tutorial page implement

### LINKED ISSUE #33: Tutorial Page
Description:
We want a tutorial page that plays like a slideshow to let user know how to use our app
We want to move the help information from settings page where it does not belong

### PR DESCRIPTION
**Pull request by cbj99**

### CODE REVIEW

File: source/scripts/tutorial_page.js [hunk 1]
```diff
@@ -0,0 +1,91 @@
+// Master DOManipulator v2 ------------------------------------------------------------
```
Comment (friidryce) [inline source/scripts/tutorial_page.js:1]: Commenting could use a bit of cleanup, perhaps some more headers and non-inline comments for better readability

File: source/styles/main.css [hunk 2]
```diff
@@ -119,6 +124,38 @@ aside {
...
+.info-button {
+    width: 40px;
+    height: 40px;
+    background: transparent;
+    border-radius: 15px;
+    border: 0;
+    margin-top: 53px;
+    margin-bottom: auto;
+    position: relative;
+    top: -212px;
```
Comment (friidryce) [inline source/styles/main.css:140]: Could be refactored into constants instead of current "magic numbers"

File: source/styles/tutorial_page.css [hunk 3]
```diff
@@ -0,0 +1,206 @@
+* {
```
Comment (friidryce) [inline source/styles/tutorial_page.css:1]: Constant refactoring doable, I understand this was kind of a rapid prototype

File: source/tutorial_page.html [hunk 4]
```diff
@@ -0,0 +1,149 @@
...
+
+    <link rel="stylesheet" href="./styles/tutorial_page.css">
+
+    <link rel="icon" type="image/png" href="./favicon.ico">
+
+
+</head>
+
+<body>
+
```
Comment (friidryce) [inline source/tutorial_page.html:20]: A bit of divitis, items could perhaps be refactored into figure/figcaption elements

**Review by friidryce — APPROVED** [review]
Good changes, some possible refactoring specified in comments, overall good work!

**Review by jinbest17 — APPROVED** [review]
Great job on implementing the tutorial page. We will update the screenshots next sprint.

**Pull request merged**